dpdata/__init__.py

```diff
@@ -9,3 +9,16 @@
     from ._version import version as __version__
 except ImportError:
     from .__about__ import __version__
+
+# BondOrder System has dependency on rdkit
+try:
+    import rdkit
+    USE_RDKIT = True
+except ModuleNotFoundError:
+    USE_RDKIT = False
+    print("WARNING: No rdkit is installed, 'BondOrderSystem' cannot be used")
```

Comment:
It's not a good idea to show this message to users who don't use this module.

Reply:
OK. I'll remove this.